src/scripts/features/searchbar.ts

```diff
@@ -415,3 +417,12 @@ function focusSearchbar() {
 		domsearchbar?.focus()
 	}
 }
+const searchBar = document.getElementById("searchbar") as HTMLInputElement;
```

Comment:
Already defined as `domsearchbar` line 33

Reply:
I'm sorry but i'm not a programmer. what am i supposed to do exactly? 😅